src/utils.py

```diff
@@ -2,10 +2,12 @@
 import os
 import re
 import shutil
+import uiautomator2 as u2
 from datetime import datetime
 from random import randint
 from time import sleep
 
+
```

Comment:
Remove extra blank line.

Reply:
There are a lot of extra spaces, it's not my code :)